### PR TITLE
fix(cmd): Add message about deprecated chart

### PR DESCRIPTION
**Pull request by kvendingoldo**
Partial #3724

### CODE REVIEW

**Comment by bacongobbler** [comment]
Hi @kvendingoldo. Thanks for the PR.

Because we're in the final release candidate stages before we release 3.0.0, we are only looking at major regressions or bug fixes fixes. Since this is a new feature, we will take a closer look at this feature for Helm 3.1.0 - after 3.0.0 has been released. Just letting you know that it'll take a while before we get around to reviewing this. Thanks!

File: cmd/helm/install.go [hunk 1]
```diff
@@ -181,6 +183,10 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 		return nil, err
 	}
 
+	if chartRequested.Metadata.Deprecated {
+		fmt.Fprintln(os.Stderr, "WARNING: Current chart is deprecated")
```
Comment (hickeyma) [inline cmd/helm/install.go]: os.Stderr --> out

File: cmd/helm/upgrade.go [hunk 2]
```diff
@@ -127,6 +128,10 @@ func newUpgradeCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 				}
 			}
 
+			if ch.Metadata.Deprecated {
+				fmt.Fprintln(os.Stderr, "WARNING: Current chart is deprecated")
```
Comment (hickeyma) [inline cmd/helm/upgrade.go]: os.Stderr --> out

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks for PR @kvendingoldo. Change on the writer to use.

**Comment by kvendingoldo** [comment]
@hickeyma I fixed your comments.

File: cmd/helm/install.go [hunk 3]
```diff
@@ -181,6 +182,10 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 		return nil, err
 	}
 
+	if chartRequested.Metadata.Deprecated {
+		fmt.Fprintln(out, "WARNING: Current chart is deprecated")
```
Comment (hickeyma) [inline cmd/helm/install.go]: Current chart is deprecated --> This chart is deprecated

File: cmd/helm/upgrade.go [hunk 4]
```diff
@@ -127,6 +127,10 @@ func newUpgradeCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 				}
 			}
 
+			if ch.Metadata.Deprecated {
+				fmt.Fprintln(out, "WARNING: Current chart is deprecated")
```
Comment (hickeyma) [inline cmd/helm/upgrade.go]: Current chart is deprecated --> This chart is deprecated

**Review by hickeyma — CHANGES_REQUESTED** [review]
@kvendingoldo Thanks for the update. Small nit on the text.

File: cmd/helm/upgrade.go (hunk 4 above)
Comment (kvendingoldo) [reply cmd/helm/upgrade.go]: Fixed

File: cmd/helm/install.go (hunk 3 above)
Comment (kvendingoldo) [reply cmd/helm/install.go]: Fixed

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @kvendingoldo

**Comment by technosophos** [comment]
picked into release-3.0

**Pull request merged**